src/worker.ts

```diff
@@ -36,15 +36,15 @@ async function getHandler (filename : string) : Promise<Function | null> {
   try {
     handler = await importESM(pathToFileURL(filename).href);
     if (typeof handler !== 'function') {
-      handler = (handler as any).default;
+      handler = await (handler as any).default;
     }
   } catch {}
   if (typeof handler !== 'function') {
     // With our current set of TypeScript options, this is transpiled to
     // `require(filename)`.
-    handler = await import(filename);
+    handler = await (await import(filename));
```

Comment:
the double await here is... interesting... not 100% sure we need it ... going to keep testing

Reply:
Yep, looks like it's not needed... ;-)